consumers.go

```diff
@@ -2,40 +2,60 @@ package main
 
 import (
 	"encoding/json"
+	"fmt"
 	"log"
 
 	"github.com/olivere/elastic"
 )
 
+// Consumer defines an interface to be implemented by various consumers
 type Consumer interface {
 	Consume(<-chan Record, chan<- bool)
 }
 
-type ESConsumer struct {
+type esConsumer struct {
```

Comment:
Why change this to not be exported? I know it's all in the same module now, but it's not crazy to imagine we might split things up later.

Reply:
I found the exporting without a clear need was...unnecessary. I'm happy to change them back though.

Reply:
I'd say it's more about intent. I'd rather they be exported.

Reply:
Sure thing. I'll switch them back as this wasn't required for the code to work as intended. I'd like to discuss this further in a future dev meeting though.